docs/using-cli.md

```diff
@@ -14,6 +15,34 @@
 * [Clear cache](#clear-cache)
 * [Check version](#check-version)
 
+## Notes on entity files written in typescript
+This CLI tool is written in javascript and to be run on node. If your entity files are in typescript, you will need to transpile them to javascript before using CLI. You may skip this section if you only use javascript.
+
+You may setup ts-node in your project to ease the operation as follows:
+
+Install ts-node globally:
```

Comment:
I would vote for using `npx ts-node` instead

Reply:
never used `npx` on my own

Reply:
It worth trying